File: Cryptographic_Backend/src/main/java/cb/backend/methods/DeleteKeyPairMethod.java
package cb.backend.methods;

import cb.backend.Method;

public abstract class DeleteKeyPairMethod implements Method {
    public static class Args {
        final long handler;

        public Args(long handler) {
            this.handler = handler;
        }
    }

    protected final long handler;

    public DeleteKeyPairMethod(Args args) {
        this.handler = args.handler;
    }

}

File: CryptographicFrontend/communication/DeleteKeyPairMethod.cpp
#include "DeleteKeyPairMethod.h"
#include "ResponseMessage.h"
#include "Argument.h"
using namespace communication;

DeleteKeyPairMethod::DeleteKeyPairMethod ( std::string handler )
    : Method ( "DeleteKeyPair" )
{
    addArgument ( "handler", handler );
}

ResponseMessage DeleteKeyPairMethod::parseResponse ( const Json::Value& value )
{
    return ResponseMessage();
}
// kate: indent-mode cstyle; replace-tabs on; 
